src/app/(tools)/coverLetter/page.tsx

```diff
@@ -0,0 +1,285 @@
+"use client";
```

Comment:
export new Metadata for this page, specific to this tool,
would help us in SEO, also adds to the tab title, easier for the user to recognize

Reply:
you will run into a rendering issue when you add it, try to see how you can rectify it :)